zerver/tests/test_user_groups.py

```diff
@@ -1568,7 +1577,7 @@ def test_query_counts(self) -> None:
             for i in range(50)
         ]
 
-        with self.assert_database_query_count(5):
+        with self.assert_database_query_count(9):
```

Comment:
I'm not entirely sure how significant this query count increase is because of the new default group and what should be done about it.